tests/instruments/hp/test_hp11713a.py

```diff
@@ -0,0 +1,96 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2023 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+from pymeasure.test import expected_protocol
+
+from pymeasure.instruments.hp.hp11713a import HP11713A, Attenuator_110dB
+
+import pytest
+
+
+class TestHP11713A:
+    @pytest.mark.parametrize("channel", list(range(0, 9)))
+    def test_channels(self, channel):
+        with expected_protocol(
+                HP11713A,
+                [("A{channel}".format(channel=channel), None),
```

Comment:
Just for your information:
`"A{channel}".format(channel=channel)` is equal to the f-string `f"A{channel}"`.
These f-strings make it easier to understand (if there are several replacements), which one is which, you can even add calculations: `f"{3+4}" == "7"`

Reply:
Thanks for the info !